core/platform/email/mailgun_email_services_test.py

```diff
@@ -19,14 +19,19 @@
 from core.platform.email import mailgun_email_services
 from core.tests import test_utils
 import feconf
+import requests
 
 
 class EmailTests(test_utils.GenericTestBase):
     """Tests for sending emails."""
 
-    def test_sending_email(self):
-        # Emails are not sent if the CAN_SEND_EMAILS setting is not turned on.
-        # mailgun api key and domain name are required to use API.
+    def test_sending_email_exceptions(self):
```

Comment:
This name isn't clear to me. Also, if they are 2 separate logical tests, can we split this into two tests?  Maybe test_sending_email_raises_exception_for_missing_api_key is a better name.

Reply:
This is from the old file but the merged logic is still in my submitted PR. I will be making another PR soon with your suggested changes (descriptive function names, splitting tests, etc.)